### DISCUSSION TITLE
SVG Icon doesn't render on Iphone (safari)

### DISCUSSION THREAD

**Discussion by ketnek** [neuefische/questions#587] [Web] [answered]
Hey Guys,

I have a little issue with the react-icons library. The implemented icon is shown everywhere, but on iphone using safari the icon is not visible. The icon is contained by an button-tag and the button works fine on ios. Only the icon is not there.
Here is the link to my actual branch: [Link](https://github.com/ketnek/capstone-project/tree/feature/destination-search). 
 
![Screenshot 2023-05-02 at 09 50 18](https://user-images.githubusercontent.com/113082466/235615234-a50cafeb-a9eb-4794-a0bc-683d02dfc7ba.png)

I have searched the web, the discussion board and asked chat gpt but without success.

Thank's for your help.

**Comment by ThomasKirst** [comment, marked as answer]
Hi Kevin,

I found that your search icon SVG uses `currentColor` for the stroke and fill attributes.
But the parent element (`button`} does not provide such a color style property that could be inherited.

You should add a color property to your button as I did quickly in the inspector:

<img width="1167" alt="Screenshot 2023-05-02 at 12 02 00" src="https://user-images.githubusercontent.com/4458383/235638004-ac67f1b0-e3c3-451b-9331-7b427f6c3ad3.png">

Hope this helps!
– Thomas

**Reply by ketnek** [reply]
Thank's Thomas, that fixed my issue👍🏻
